components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```
@@ -132,14 +136,18 @@ def handle_job(sql_adapter: SQL_Adapter, clp_io_config: ClpIoConfig, no_progress
-    clp_config: CLPConfig, logs_to_compress: List[str], parsed_args: argparse.Namespace
-) -> typing.Union[S3InputConfig, FsInputConfig]:
+    clp_config: CLPConfig,
+    logs_to_compress: List[str],
+    parsed_args: argparse.Namespace,
+    dataset: Optional[str],
+) -> Union[S3InputConfig, FsInputConfig]:
     input_type = clp_config.logs_input.type
 
     if InputType.FS == input_type:
         if len(logs_to_compress) == 0:
             raise ValueError(f"No input paths given.")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)